provision-vm/azuredeploy.json

```diff
@@ -225,11 +239,17 @@
                 "autoUpgradeMinorVersion": true,
                 "settings": {
                     "fileUris": [
-                        "https://raw.githubusercontent.com/Azure-Samples/openhack-devops-proctor/master/provision-vm/proctorVMSetup.sh"
+                        "https://raw.githubusercontent.com/Azure-Samples/openhack-devops-proctor/feature/selfguided/provision-vm/proctorVMSetup.sh"

```

Comment:
is this the right repo url?  Assume once in you will need to send another PR to set to master?